packages/cli/src/daemon-config.ts

```diff
@@ -420,6 +451,12 @@ export class DaemonConfig {
   static async fromFile(filepath: URL): Promise<DaemonConfig> {
     const content = await readFile(filepath, { encoding: 'utf8' })
     const config = DaemonConfig.fromString(content)
+    // TODO: Replace with commented lines below when we fully deprecate IP address access to mainnet
+    if (!config.node) config.anchor.authMethod = AnchorServiceAuthMethods.NONE
```

Comment:
@pawartur Here we allow the node to start even if the private seed url is not in the config